packages/aws-rfdk/lib/lambdas/nodejs/configure-spot-event-plugin/test/handler.test.ts

```diff
@@ -480,6 +513,57 @@ describe('SEPConfiguratorResource', () => {
         .rejects
         .toThrowError(/Failed to save Spot Event Plugin Configurations/);
     });
+
+    test('throw when cannot save group collection', async () => {
```

Comment:
```suggestion
    test('throw when cannot save collection', async () => {
```

I think we can generalize this test and just use a variable `const collectionType = CollectionType.Group` as our test case. Then we can assert the error message contains the `collectionType` we passed in.

Alternatively, we can parameterize this test for all the values in `CollectionType` using something like `test.each`:

```ts
test.each(
  Object.values(CollectionType)
)('throws when collection fails to save (%s)', async (ct: CollectionType) => {
  // Test code here
});
```